### DISCUSSION TITLE
Missing releases?

### DISCUSSION THREAD

**Discussion by Stanzilla** [nklayman/vue-cli-plugin-electron-builder#1311] [Q&A] [answered]
Hey there, I noticed the releases rc5 and rc6 are missing on the repo, is there a problem with them that made you delete them? Just wondering because I'm on rc6 and would like to know if I should downgrade.

**Comment by nklayman** [comment, marked as answer]
No need to downgrade, the releases are just fine. I just forgot to make a release on github. For smaller releases I often don't create them, you can use the commit log as a changelog instead.
